mock-abis/src/main/java/io/mosip/proxy/abis/service/impl/ProxyAbisConfigServiceImpl.java

```diff
@@ -15,54 +15,67 @@
 @Service
 public class ProxyAbisConfigServiceImpl implements ProxyAbisConfigService {
 
-    @Autowired
-    ProxyAbisInsertRepository proxyabis;
-
-    @Autowired
-    ProxyAbisBioDataRepository proxyAbisBioDataRepository;
-
-    @Autowired
-    private ExpectationCache expectationCache;
-
-    /**
-     * set this flag to false then we will not check for duplicate we will always return unique biometric
-     */
-    @Value("${abis.return.duplicate:true}")
-    private boolean findDuplicate;
-
-    public Boolean getDuplicate(){
-        return findDuplicate;
-    }
-    public void setDuplicate(Boolean d){
-        findDuplicate = d;
-    }
-
-    public Map<String, Expectation> getExpectations(){
-        return expectationCache.get();
-    }
-
-    public void setExpectation(Expectation exp){
-        expectationCache.insert(exp);
-    }
-
-    public void deleteExpectation(String id){
-        expectationCache.delete(id);
-    }
-
-    public void deleteExpectations(){
-        expectationCache.deleteAll();
-    }
-
-    public List<String> getCachedBiometrics(){
-        return proxyAbisBioDataRepository.fetchAllBioData();
-    }
-
-    public List<String> getCachedBiometric(String hash){
-        return proxyAbisBioDataRepository.fetchByBioData(hash);
-    }
-
-    public void deleteAllCachedBiometrics(){
-        proxyAbisBioDataRepository.deleteAll();
-        proxyabis.deleteAll();
-    }
+	@Autowired
+	ProxyAbisInsertRepository proxyabis;
+
+	@Autowired
+	ProxyAbisBioDataRepository proxyAbisBioDataRepository;
+
+	@Autowired
+	private ExpectationCache expectationCache;
+
+	/**
+	 * set this flag to false then we will not check for duplicate we will always
+	 * return unique biometric
+	 */
+	@Value("${abis.return.duplicate:true}")
+	private boolean findDuplicate;
+
+	private boolean newDuplicate;
+
+	public Boolean getDuplicate() {
+		return findDuplicate;
+	}
+
+	public void setDuplicate(Boolean d) {
+		findDuplicate = d;
+	}
+
+	public Boolean getNewDuplicate() {
```

Comment:
this is not what we discussed. Where are we using this new duplicate? why did we introduce this new duplicate?